manubot/command.py

```diff
@@ -0,0 +1,74 @@
+"""
+Manubot's command line interface
+"""
+import argparse
+import logging
+import sys
+import warnings
+
+import errorhandler
+
+import manubot
+from manubot.cite import (
+    cli_cite,
+    configure_cite_argparser,
+)
+from manubot.process import (
+    cli_process,
+    configure_process_argparser,
+)
+
+
+def parse_arguments():
+    """
+    Read and process command line arguments.
+    """
+    parser = argparse.ArgumentParser(description='Manubot: the manuscript bot for scholarly writing')
+    parser.add_argument('--version', action='version', version=f'v{manubot.__version__}')
+    parser.add_argument('--log-level', default='WARNING',
+                        choices=['DEBUG', 'INFO', 'WARNING', 'ERROR', 'CRITICAL'],
+                        help='Set the logging level for stderr logging')
+    subparsers = parser.add_subparsers()
+    # Require specifying a sub-command
+    subparsers.required = True  # https://bugs.python.org/issue26510
+    subparsers.dest = 'command'  # https://bugs.python.org/msg186387
```

Comment:
I don't what know we can do about it, but I do not find the error message

> manubot: error: the following arguments are required: command

very intuitive.  Perhaps we could add a line to the `description` about specifying a command `process` or `cite`.